miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/ValidationUtils.java

```diff
@@ -1,33 +1,23 @@
 package uk.ac.bbsrc.tgac.miso.service.impl;
 
-import static uk.ac.bbsrc.tgac.miso.core.util.LimsUtils.isStringEmptyOrNull;
+import uk.ac.bbsrc.tgac.miso.core.data.*;
+import uk.ac.bbsrc.tgac.miso.core.service.ProviderService;
+import uk.ac.bbsrc.tgac.miso.core.service.exception.ValidationError;
+import uk.ac.bbsrc.tgac.miso.core.service.exception.ValidationException;
+import uk.ac.bbsrc.tgac.miso.core.service.naming.NamingScheme;
+import uk.ac.bbsrc.tgac.miso.core.service.naming.validation.ValidationResult;
+import uk.ac.bbsrc.tgac.miso.core.util.WhineyFunction;
 
 import java.io.IOException;
 import java.math.BigDecimal;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.Collection;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
```

Comment:
Looks like you removed a bunch of imports that are still needed?

Reply:
I used Optimize Imports, it imports java.util.* instead

Reply:
Oh, didn't see that. Please do not use that option though as the Eclipse settings in the repository do not use it, and are meant to minimize the diff between users

Reply:
Bump

Reply:
expanded the asterisks back out